src/fsharp/ErrorLogger.fs

```diff
@@ -215,7 +215,7 @@ type PhasedError = { Exception:exn; Phase:BuildPhase } with
 [<System.Diagnostics.DebuggerDisplay("{DebugDisplay()}")>]
 type ErrorLogger(nameForDebugging:string) = 
     abstract ErrorCount: int
-    // the purpose of the 'Impl' factoring is so that you can put a breakpoint on the non-Impl code just below, and get a breakpoint for all implementations of error loggers
+    // The purpose of the 'Impl' factoring is so that you can put a breakpoint on the non-Impl code just below, and get a breakpoint for all implementations of error loggers
```

Comment:
The 'Impl' factoring enables a developer to place a breakpoint at the non-Impl code just below and get a breakpoint for all error logger implementations.